libs/ledgerjs/packages/hw-app-exchange/src/Exchange.ts

```diff
@@ -12,7 +14,11 @@ export const enum ExchangeTypes {
   Swap = 0x00,
   Sell = 0x01,
   Fund = 0x02,
+  SwapNg = 0x03,
```

Comment:
What means Ng sufix?

Reply:
It is for Next Gen :P
It comes from the firmware team, for consistency I keep their naming.